doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -261,19 +327,60 @@ There are four sorts :g:`SProp`, :g:`Prop`, :g:`Set`  and :g:`Type`.
 
 More on sorts can be found in Section :ref:`sorts`.
 
+.. _fix:
+
+Fix and Cofix
+-------------
+
+.. insertgram single_fix fix_body
+
+.. productionlist:: coq
+   single_fix : fix `fix_body`
+              : cofix `fix_body`
+   fix_body_list : `fix_body_list` with `fix_body`
+                 : `fix_body`
+   fix_body : `ident` `binders_fixannot` `type_cstr` := `term`
+
 .. _binders:
 
 Binders
 -------
 
-Various constructions such as :g:`fun`, :g:`forall`, :g:`fix` and :g:`cofix`
-*bind* variables. A binding is represented by an identifier. If the binding
-variable is not used in the expression, the identifier can be replaced by the
-symbol :g:`_`. When the type of a bound variable cannot be synthesized by the
-system, it can be specified with the notation :n:`(@ident : @type)`. There is also
-a notation for a sequence of binding variables sharing the same type:
-:n:`({+ @ident} : @type)`. A
-binder can also be any pattern prefixed by a quote, e.g. :g:`'(x,y)`.
```

Comment:
Did you remove this explanation because it seemed like a textual repetition of what is presented in the grammar? Maybe we can simplify the explanation, but removing it entirely seems too much.

Reply:
Unintentional